docs/rule/require-presentational-children.md

```diff
@@ -53,6 +53,13 @@ This rule **allows** the following:
 
 If violations are found, remediation should be planned to either add `role="presentation"` to the descendants as a quickfix. A better fix is to not use semantic descendants.
 
+## Configuration
+
+ The following values are valid configuration:
+
+* object -- An object with the following keys:
+  * `nonSemanticTags` -- An array of additional tags that should be considered presentation
```

Comment:
Probably we should name this `additionalNonSemanticTags` similar to `additionalInteractiveTags` in https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-invalid-interactive.md#configuration